rules.md

```diff
@@ -122,17 +99,13 @@ All Bounties start as a Concept that is [proposed](#31-proposing-concepts) to th
 
 Concepts are proposals for some work that the Incubator should fund as a Bounty, that provide enough detail about the value proposition and requirements of the work to enable Contributors to complete it.
 
-[Admins](#4-admins) create a Bounty for each Concept they Accept that they then Own, meaning they have the rights to manage and earn commission from its delivery.
-
-Admins can [Accept](#435-processing-task-claims) any Concept that meets our [Concept Acceptance Criteria](#411-concept-acceptance-criteria) on a first-come first-serve basis.
-
-Once accepted, the user who created the Concept is automatically assigned as a [Stakeholder](#25-bounty-stakeholders) of the corresponding bounty.
+[Strategists](#46-strategists) create a Bounty for each Concept they accept that they then own, meaning they have the rights to manage and earn commission from its delivery.  They [accept](#434-processing-task-claims) Concepts that meet our [Concept Acceptance Criteria](#411-concept-acceptance-criteria) on a first-come first-serve basis.  Once accepted, the user who created the Concept is assigned as a [Stakeholder](#25-bounty-stakeholders) of the corresponding bounty.
```

Comment:
should a stakeholder be listed as one of the user types and defined at the opening of this section? (line 59)